tests/test_benchmarks/test_zhu2019.py

```diff
@@ -0,0 +1,82 @@
+from pathlib import Path
+
+import numpy as np
+import pytest
+from pytest import approx
+
+from brainio.assemblies import BehavioralAssembly
+from brainscore import benchmark_pool
+from brainscore.benchmarks.zhu2019 import DATASETS
+from tests.test_benchmarks import PrecomputedFeatures
+
+'''
+Tests all use a precomputed ceiling. See Zhu2019.py for more details.
+'''
+
+
+@pytest.mark.private_access
+class TestZhu2019:
+    def test_count(self):
+        assert len(DATASETS) == 1
+
+    @pytest.mark.parametrize('benchmark', [
+        'Zhu2019-accuracy',
+        'Zhu2019-accuracy-engineering',
+    ])
+    def test_in_pool(self, benchmark):
+        assert benchmark in benchmark_pool
+
+    @pytest.mark.parametrize('benchmark, expected_ceiling', [
+        ('Zhu2019-accuracy', approx(0.8113, abs=0.0001)),
+        ('Zhu2019-accuracy-engineering', approx(1.0000, abs=0.0001)),
+    ])
+    def test_benchmark_ceiling(self, benchmark, expected_ceiling):
+        benchmark = benchmark_pool[benchmark]
+
+        if "engineering" in benchmark.identifier:
+            assembly = benchmark._assembly
+            ceiling = benchmark._ceiling(assembly)
+            assert ceiling[(ceiling['aggregation'] == 'center')] == expected_ceiling
+        else:
+            ceiling = benchmark._ceiling
+            assert ceiling == expected_ceiling
+
+    @pytest.mark.parametrize('benchmark, model, expected_raw_score', [
+        ('Zhu2019-accuracy', 'alexnet', approx(0.1880, abs=0.0001)),
+        ('Zhu2019-accuracy-engineering', 'alexnet', approx(0.470, abs=0.001)),
+        ('Zhu2019-accuracy', 'resnet-18', approx(0.204, abs=0.0001)),
+        ('Zhu2019-accuracy-engineering', 'resnet-18', approx(0.506, abs=0.001)),
+    ])
+    def test_model_raw_score(self, benchmark, model, expected_raw_score):
+
+        # load features
+        precomputed_features = Path(__file__).parent / f'{model}-{benchmark}.nc'
+        benchmark = benchmark_pool[benchmark]
+        precomputed_features = BehavioralAssembly.from_files(file_path=precomputed_features)
+        precomputed_features = PrecomputedFeatures(precomputed_features,
+                                                   visual_degrees=8.0,  # doesn't matter, features are already computed
+                                                   )
+        score = benchmark(precomputed_features)
+        raw_score = score.raw
+        assert raw_score[0] == expected_raw_score
```

Comment:
```suggestion
```
redundant with line 65 (and using more error-prone positional indexing instead of label based)